alert-policies/lampy/TransactionErrors.yml

```diff
@@ -0,0 +1,36 @@
+name: Transaction Errors
+
+description: |+
+  This alert is triggered when the transactions fail more than 10% of the time in 5 minutes.
+type: STATIC
+nrql:
+  query: "SELECT percentage(count(*), WHERE error is true)  FROM Transaction"
```

Comment:
can be changed to: `query: "SELECT percentage(count(*), WHERE error is true) FROM Transaction"